include/dlaf/matrix/print_csv.h

```diff
@@ -72,4 +77,4 @@ void print(format::csv, std::string sym, Matrix<const T, Device::CPU>& mat,
   }
 }
 }
-}
+}
```

Comment:
Missing new line at the and of the file
Why `check_format` is not failing?

Reply:
Does `clang-format` add newlines?

Reply:
I thought yes, but it only removes extra lines.
This feature is available only from clang 16...

Reply:
It seems like the checks finished now... and `check_format` is clean. @gulivarese could you remove this change since it's not part of adding missing includes?

Reply:
@rasolca now it has finished in 57 seconds.